app/views/hiring_staff/vacancies/documents/show.html.haml

```diff
@@ -13,6 +13,7 @@
 
       = f.govuk_file_field :documents,
         label: { text: t('jobs.upload_file'), size: 's' },
+        accept: '.doc, .xls, .ppt, .pdf, image/jpeg, image/png, video/mp4',
```

Comment:
Shouldn't this have `.docx` and `.xlsx` as well (is there a `.pptx`?).

Reply:
There is, and I asked the question but no response. Will add now 👍